tests/test_json.py

```diff
@@ -186,6 +187,14 @@ def test_config() -> None:
             "Profile": "profile",
             "QemuArgs": [],
             "QemuCdrom": false,
+            "QemuDrives": [
+                {
+                    "directory": "/foo/bar",
+                    "id": "abc",
+                    "options": "abc,qed",
+                    "size": 200
+                }
```

Comment:
Let's add the directory null case for completeness
```suggestion
                {
                    "directory": "/foo/bar",
                    "id": "abc",
                    "options": "abc,qed",
                    "size": 200
                },
                {
                    "directory": null,
                    "id": "def",
                    "options": "",
                    "size": 123456
                }
```